1-exercises/I-string-replace/exercise.js

```diff
@@ -13,7 +13,10 @@
 let story =
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
 
-let result = story.replace("", "");
+let result = story.replace(/dogs/g, "cats");
+result = result.replace("0", "00000");
```

Comment:
Good solution apart from the typo ; `00000` instead of `100000`

Reply:
Noted! Thanks